src/middlewared/middlewared/plugins/system.py

```diff
@@ -568,11 +573,30 @@ async def info(self):
             'license': await self.middleware.run_in_thread(self._get_license),
             'boottime': datetime.fromtimestamp(psutil.boot_time()),
             'datetime': datetime.utcnow(),
-            'timezone': (await self.middleware.call('datastore.config', 'system.settings'))['stg_timezone'],
+            'birthday': birthday_date,
+            'timezone': timezone_setting,
             'system_manufacturer': manufacturer,
             'ecc_memory': ecc_memory,
         }
 
+    def _convert_datetime(self, response, timezone_setting=timezone.utc):
+        return datetime.fromtimestamp(response.tx_time, timezone_setting)
```

Comment:
This method is useless as its not used nor meant to me used anywhere, just add another thing to the stack call.